### ISSUE TITLE
Test Pilot Addon Doesn't Unbadge

### ISSUE THREAD

**Issue by marniepw** [mozilla/testpilot#3877]
Was in a meeting sharing my screen, John called out my Test Pilot addon because it still has the blue ! new experiment badge.
We haven't launched an experiment since Advance and that's been over two weeks ago; he said that should go away after two weeks.

Mac. Nightly: 64.0a1 (2018-09-14) 

<img width="405" alt="screen shot 2018-09-20 at 1 20 50 pm" src="https://user-images.githubusercontent.com/33703387/45844976-b0cd4300-bcd8-11e8-906d-314d56d34487.png">

**Comment by lmorchard** [comment]
Probably related to / could be investigated with https://github.com/mozilla/testpilot/issues/3604

**Comment by meandavejustice** [comment]
@marniepw Can you add anymore info that can help in reproducing?

**Comment by marniepw** [comment]
@meandavejustice not really. It's still badged. It "turned on" when we rolled out Advance. I never clicked on it. It's still badged currently. I've been to the Test Pilot website to look at experiments, but I've never clicked on the icon.

**Comment by SoftVision-PaulOiegas** [comment]
@meandavejustice If I recall right (not sure), we had a rule something like `if the user does not click the button in 2 weeks, the badge is removed`. Maybe something broke there?

**Comment by meandavejustice** [comment]
after reviewing the code, this definitely shouldn't be happening. The only thing I can think of is that something errored in the addon, preventing it from doing the check to update the badge on an interval. @marniepw, if you still have the badge would you mind hopping into a vidyo session with me sometime this week and we can maybe debug together? I'd like to take a look at your browser console on startup.

**Comment by johngruen** [comment]
Let's keep an eye out when we do a stage release, i'm content to close if we don't see anything weird there
